troubleshooting/common-issues.md

```diff
@@ -64,8 +64,8 @@ _**Solution:**_ In such a case, we're providing you with a fallback to normal pa
 To enable it, first **stop and re-start the yagna daemon** and then run:
 
 ```text
-yagna payment init -r --driver=ngnt
-yagna payment status --platform=NGNT
+yagna payment init -r --driver=erc20
+yagna payment status --network=rinkeby
```

Comment:
I'm unsure about this. Seems there is no more `platform` parameter if I'm correct. Please advice.

Reply:
you are correct, `--platform` is split into `--driver` `--network` `--token`